verifyFiles.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
+
+# #verifyFiles.py
+#
+#Script to check whether the output files generated across 
+#subject folders in different conditions are the same or not.
+#The conditions on which the script will check the match between two files are 
+#filename, size, content, modification time, distance etc.
+#
+#
+#
+#
+# ##Maintainters
+#
+# * Big Data Lab Team, Concordia University.
+# * email : tristan.glatard@concordia.ca, laletscaria@yahoo.co.in
+#
+#
+
+import os
+import sys
+import subprocess
+import argparse,textwrap
+import hashlib
+from collections import defaultdict,OrderedDict
+
+#study_folder_details_dict_list is a list for storing ordered dictionaries 
+#containing the details regarding the files of  individual subjects.
+study_folder_details_dict_list=[]
```

Comment:
I don't think this should be declared globally. Should be declared in main and passed as arguments to the subsequent functions (which is done already).